ee/clickhouse/sql/funnels/funnel.py

```diff
@@ -1,16 +1,19 @@
 FUNNEL_SQL = """
+SELECT id, max_step FROM (
     SELECT 
         pid.person_id as id,
         windowFunnel(6048000000000000)(toUInt64(toUnixTimestamp64Micro(timestamp)),
             {steps}
-        )
+        ) as max_step
     FROM 
-            events
+        events
     JOIN (
         SELECT person_id, distinct_id FROM person_distinct_id WHERE team_id = %(team_id)s
     ) as pid
     ON pid.distinct_id = events.distinct_id
     WHERE
         team_id = %(team_id)s {filters} {parsed_date_from} {parsed_date_to}
+        AND event IN %(events)s
     GROUP BY pid.person_id
```

Comment:
not sure if this would be faster but could be just `GROUP BY pid.person_id HAVING max_step > 0`

Reply:
Ah yes this maybe would have worked in my previous iteration, but i've changed the query a lot. I did test putting it in that one but it was a little slower